rfcs/text/0000-metadata-integration-test.md

```diff
@@ -48,6 +48,7 @@ Data with the new schema version should not be uploaded into the `integration` o
 
 * May introduce another deployment stage
 * Treats metadata schema integration differently from other DCP projects
+* Requires test data to that use the new schema features
 
 ## Alternative design
 
```

Comment:
Please add a title to the first alternative design to differentiate the two. Perhaps `### All projects must PR before merging into integration environment`